frontend/src/pages/primary/index.tsx

```diff
@@ -111,6 +111,10 @@ export default function Primary() {
         }
     }, [nav])
 
+    useEffect(() => {
+        setAudioBlobPart(undefined);
+    }, [text])
+
```

Comment:
it is, but i am combining all the relevant state stuff in that file. It would be possible somehow, but i wont be able to spend time on that until we finish the course